plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -263,9 +263,23 @@ read_write::push_block_results read_write::push_block(const read_write::push_blo
 }
 
 read_write::push_transaction_results read_write::push_transaction(const read_write::push_transaction_params& params) {
-   auto ptrx = db.push_transaction(params, skip_flags);
+   auto pretty_input = db.transaction_from_variant( params );
+   auto ptrx = db.push_transaction(pretty_input, skip_flags);
    auto pretty_trx = db.transaction_to_variant( ptrx );
-   return read_write::push_transaction_results{ params.id(), pretty_trx };
+   return read_write::push_transaction_results{ pretty_input.id(), pretty_trx };
+}
+
+read_write::push_transactions_results read_write::push_transactions(const vector<read_write::push_transaction_params>& params) {
```

Comment:
`const push_transactions_results& params` would match the declaration.

Reply:
I think you mean `push_transaction_params` but otherwise 👍

Reply:
yes, of course, thanks